pkg/apiserver/registry/system/controllerinfo/rest.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controllerinfo
+
+import (
+	"context"
+
+	uuid "github.com/satori/go.uuid"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/internalversion"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apiserver/pkg/registry/rest"
+
+	clusterinfo "github.com/vmware-tanzu/antrea/pkg/apis/clusterinformation/v1beta1"
+	system "github.com/vmware-tanzu/antrea/pkg/apis/system/v1beta1"
+	"github.com/vmware-tanzu/antrea/pkg/monitor"
+)
+
+// REST implements rest.Storage for ControllerInfo.
+type REST struct {
+	controllerQuerier monitor.ControllerQuerier
+}
+
+// Name of the AntreaControllerInfo resource.
+const ControllerInfoResourceName = "antrea-controller"
+
+var (
+	// uuidNamespace is a uuid.UUID type generated from a string to be
+	// used to generate uuid.UUID for system API group objects like
+	// AntreaControllerInfo.
+	// 7d077217-3a57-4128-b94b-675956c472cc was generated using
+	// uuid.NewV4() function.
+	uuidNamespace = uuid.FromStringOrNil("7d077217-3a57-4128-b94b-675956c472cc")
+
+	// UUID of the AntreaControllerInfo resource.
+	controllerInfoResourceUID = types.UID(uuid.NewV5(uuidNamespace, ControllerInfoResourceName).String())
```

Comment:
Is UUID mandatory?

Reply:
I guess no. But do you think better to add it?

Reply:
Ok removed UID.